### PR TITLE
Updated HQ descriptions to fit new limits

### PR DESCRIPTION
**Pull request by petebryan**
Required items, please complete
   
   Change(s):
   - Updated Hunting Queries in the Webshell Solution to have descriptions that meet the 255 character limit.
  - Moved longer descriptions to the new description-detailed field.

   Reason for Change(s):
   - Required to ensure correct deployment of new Webshell solution is possible.

   Version Updated:
   - Y

   Testing Completed:
   - Y

   Checked that the validations are passing and have addressed any issues that are present:
   -Y

### CODE REVIEW

**Review by v-atulyadav — COMMENTED** [review]
Hi @petebryan, hunting queries description exceeds 255 characters limit, please check for the same. Thanks

**Comment by v-atulyadav** [comment]
Hi @petebryan, Are there any updates on hunting queries with descriptions longer than 255 characters? Thanks

**Comment by petebryan** [comment]
> Hi @petebryan, hunting queries description exceeds 255 characters limit, please check for the same. Thanks

Done

File: Solutions/Web Shells Threat Protection/Hunting Queries/SpringshellWebshellUsage.yaml [hunk 1]
```diff
@@ -1,6 +1,8 @@
 id: 6911d1df-4204-43b2-a64c-3cb102551ddd
 name: Possible Webshell usage attempt related to SpringShell(CVE-2022-22965)
 description: |
+  This query searches Azure Web Application Firewall data for potential Webshell usage related to the SpringShell RCE vulnerability (CVE-2022-22965). For more information on detecting and protecting against this vulnerability, refer to Microsoft's security blog.
```
Comment (v-atulyadav) [inline Solutions/Web Shells Threat Protection/Hunting Queries/SpringshellWebshellUsage.yaml:4]: Hi @petebryan, this query description exceeds 255 characters. Thanks

Comment (petebryan) [reply Solutions/Web Shells Threat Protection/Hunting Queries/SpringshellWebshellUsage.yaml]: Done

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**